python/orca/src/bigdl/orca/learn/pytorch/training_operator.py

```diff
@@ -232,7 +232,7 @@ def _train_loop(self, iterator, info, _progress_bar, metric_meters, callbacks):
             self.global_step += 1
             if callbacks is not None:
                 for callback in callbacks:
-                    callback.on_batch_end(batch_idx)
+                    callback.on_batch_end(batch_idx, logs=metrics)
```

Comment:
If the metrics is none, we are not able to retrieve any tensorboard events?

Reply:
Yes.